swan-lake/development-tutorials/deployment-guide/deployment-guidelines-overview.md

```diff
@@ -10,7 +10,7 @@ intro: This guideline is intended to provide a comprehensive overview of best pr
 When deploying Ballerina applications in production, you can choose from several deployment types depending on your needs.
 Each option balances flexibility, control, and operational overhead based on your specific deployment types:
 
-1. [Virtual Machines or Bare Metal](/learn/virtual-machine-deployment): Provides full control over the infrastructure, making it suitable for high-performance or legacy systems requiring custom configurations or direct hardware access.
+1. [Virtual Machines or Bare Metal](/learn/monolithic-deployment): Provides full control over the infrastructure, making it suitable for high-performance or legacy systems requiring custom configurations or direct hardware access.
```

Comment:
Shouldn't we change this `Virtual Machines or Bare Metal` to `Monolithic Deployment`?
```suggestion
1. [Monolithic Deployment](/learn/monolithic-deployment): Provides full control over the infrastructure, making it suitable for high-performance or legacy systems requiring custom configurations or direct hardware access.
```